pages/dkp/kommander/2.2/legal/version-policy/index.md

```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
+
+- Tech Previews can be used in Production Environments.
```

Comment:
I don't thing we want this here.     Keep in mind that the term 'tech preview' may apply to more than just insights over time.   And my understanding is that engineering is not agreeing to provide the current product (insights) with the SLA's necessary to support 'production environments'.   Certainly support is not.